hoomd/md/ForceComposite.cc

```diff
@@ -8,6 +8,10 @@
 #include <sstream>
 #include <string.h>
 
+#ifndef __HIPCC__
+#include <pybind11/stl.h>
+#endif
```

Comment:
```suggestion
#include <pybind11/stl.h>
```
Check not needed in `.cc` files.